spool.py

```diff
@@ -90,70 +91,85 @@ def spool_render(rman_version_short, rib_files, denoise_files, denoise_aov_files
                                    frame_num, False, 3)
             for in_name, out_name, options in frame_texture_cmds[frame_num]:
                 in_name = bpy.path.abspath(in_name)
-                out_name = os.path.join(rpass.paths['texture_output'], out_name)
+                out_name = os.path.join(
+                    rpass.paths['texture_output'], out_name)
                 txmake_task(f, "TxMake %s" % os.path.split(in_name)
                             [-1], in_name, out_name, options, 4)
             end_block(f, 3)
 
         # render frame
-        cmd_str = ['prman', '-Progress', '-cwd', cdir, '-t:%d' %
-                   rm.threads, rib_files[frame_num - frame_begin]]
+        cmd_str = ['prman', '-Progress', '-cwd',
+                   cdir, rib_files[frame_num - frame_begin]]
```

Comment:
Yes. Though it might be better as a custom render command.

Sent from my iPhone

On Aug 25, 2016, at 5:17 PM, bsavery <notifications@github.com<mailto:notifications@github.com>> wrote:


In spool.py<https://github.com/bsavery/PRMan-for-Blender/pull/350#discussion_r76335414>:

>                  txmake_task(f, "TxMake %s" % os.path.split(in_name)
>                              [-1], in_name, out_name, options, 4)
>              end_block(f, 3)
>
>          # render frame
> -        cmd_str = ['prman', '-Progress', '-cwd', cdir, '-t:%d' %
> -                   rm.threads, rib_files[frame_num - frame_begin]]
> +        cmd_str = ['prman', '-Progress', '-cwd',
> +                   cdir, rib_files[frame_num - frame_begin]]


Did the threads number get deleted on purpose?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/bsavery/PRMan-for-Blender/pull/350/files/c981a39fbab793f5c468046be847735320ac63bc#r76335414>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AM5RFK7MVyBuu7Gj4pdGqRt4GQC5JxrSks5qjhSCgaJpZM4Jp2CY>.